examples/kmm-sample/iosApp/iosApp/ContentView.swift

```diff
@@ -4,6 +4,30 @@ import shared
 struct ContentView: View {
     let calculator = Calculator.Companion()
     let greet = Greeting().greeting()
+    @State var token: LibraryCancellable? = nil
+    var counter: Counter
+    
+    class Counter: ObservableObject {
```

Comment:
As far as I can tell, this class is unneeded and could just be represented as:
`@State var count = 0`